src/main/resources/assets/computercraft/lua/rom/apis/window.lua

```diff
@@ -321,7 +321,7 @@ function create( parent, nX, nY, nWidth, nHeight, bStartVisible )
         if type( color ) ~= "number" then
             error( "bad argument #1 (expected number, got " .. type( color ) .. ")", 2 )
         elseif tHex[color] == nil then
-            error( "Invalid color", 3 )
+            error( "Invalid color (got " .. color .. ")", 2 )
```

Comment:
@lupus590 I presume the 3 was a relic from when `window.setBackgroundColor` and `window.setBackgroundColour` were separate functions which deligated to this.